public/app/themes/justice/inc/search.php

```diff
@@ -265,20 +265,22 @@ public function relevanssiParentFilter(array $hits): array
         if (!isset($wp_query->query_vars['parent'])) {
             // No parent parameter set, do nothing.
             return $hits;
+        } else {
```

Comment:
You don't need else here, the if will return, meaning no dice.

```php
if (!isset($wp_query->query_vars['parent'])) {
    // No parent parameter set, do nothing.
    return $hits;
}

$parent_id = $wp_query->query_vars['parent'];
```

A general guide I like to follow in coding: **_if else, why?_**